schema/ContainerRecipe.schema.yaml

```diff
@@ -0,0 +1,42 @@
+title: ContainerRecipe
+'@id': stencila:ContainerRecipe
+extends: SoftwareSourceCode
+role: primary
+status: experimental
+category: code
+description: The set of build instructions, typically in a text file, to generate a linux container image
+properties:
+  applicationCategory:
+    '@id': stencila:applicationCategory
+    description: The container technology (e.g., Docker or Singularity)
+    type: string
+  citation:
+    '@id': stencila:citation
+    description: One or more citations or references for the container recipe
+    type: array
+    items:
+      type: string
+  dateCreated:
+    '@id': schema:dateCreated
+    description: Date/time of creation.
+    allOf:
+      - $ref: '#/definitions/dateProperty'
+  dateModified:
+    '@id': schema:dateModified
+    description: Date/time of most recent modification.
+    allOf:
+      - $ref: '#/definitions/dateProperty'
```

Comment:
`citation`, `dateCreated`, `dateModified` and `url` do not need to be defined here because they are inherited from `CreativeWork`.

Reply:
Gotcha.